src/panels/config/core/ha-config-system-navigation.ts

```diff
@@ -41,33 +42,31 @@ class HaConfigSystemNavigation extends LitElement {
         back-path="/config"
         .header=${this.hass.localize("ui.panel.config.dashboard.system.main")}
       >
-        <ha-button-menu
-          corner="BOTTOM_START"
-          slot="toolbar-icon"
-          @action=${this._handleAction}
-        >
+        <div slot="toolbar-icon">
           <ha-icon-button
-            slot="trigger"
-            .label=${this.hass.localize("ui.common.overflow_menu")}
-            .path=${mdiDotsVertical}
+            .label=${this.hass.localize("ui.dialogs.quick-bar.title")}
+            .path=${mdiMagnify}
+            @click=${this._showQuickBar}
```

Comment:
I don't think it adds much value to add it here as there are no further sub-navigation pages. All pages you can search for are the names of the things already on your screen.

Reply:
Its more for a continuation of the config menu

Reply:
Sure. I feel like adding the search more places is only adding functionality without much negative 🤷 

But down to remove. I don't have that strong of an opinion

Reply:
Let's remove it and only leave it on the entry levels of panels (Lovelace/Config)

Reply:
done